core/workspace_svg.ts

```diff
@@ -1224,24 +1225,20 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
       // Currently does not support toolboxes in mutators.
       this.toolbox_.setVisible(isVisible);
     }
-    if (isVisible) {
-      const blocks = this.getAllBlocks(false);
-      // Tell each block on the workspace to mark its fields as dirty.
-      for (let i = blocks.length - 1; i >= 0; i--) {
-        blocks[i].markDirty();
-      }
-
-      this.render();
-      if (this.toolbox_) {
-        this.toolbox_.position();
-      }
-    } else {
+    if (!isVisible) {
```

Comment:
Why is this one not needed? Setting the workspace visibility doesn't directly change the blocks or fields, so no rerender would be triggered.

Put another way: what case was this handling before?

Reply:
I have no idea what case this was handling before. But I can't think of a reason it's needed now! I manually tested showing and hiding the workspace, and everything looks correct.

Reply:
Dug into the blame, and the render existed originally when `setVisible` was implemented.: https://github.com/google/blockly/commit/e8ab929b6ae9f1d84962c91489a5f6a04792b728 So maybe itw as necessary at some point? Or maybe it was just added cautiously.

I tested adding block to the workspace while the workspace was hidden, and that also works and renders things correctly. So I think this is safe to remove!